crates/trigger/src/lib.rs

```diff
@@ -361,27 +304,9 @@ fn decode_preinstantiation_error(e: anyhow::Error) -> anyhow::Error {
     e
 }
 
-fn key_value_file_location(app: &App) -> Option<PathBuf> {
```

Comment:
Moved to `RuntimeConfig::sqlite_db_path`. Maybe should be `key_value_file_location` instead? :shrug: